draft-ietf-taps-interface.md

```diff
@@ -1669,12 +1685,55 @@ Type:
 Default:
 : Disabled
 
-This property specifies how long to wait before deciding that a Connection has
-failed when trying to reliably deliver data to the destination. Adjusting this Property
+This property specifies how long to wait before deciding that an active Connection has
+failed when trying to reliably deliver data to the Remote Endpoint. Adjusting this Property
 will only take effect when the underlying stack supports reliability. The special value
 `Disabled` means that this timeout is not scheduled to happen. This can be a valid
 choice with unreliable data transfer (e.g., when UDP is the underlying transport protocol).
+See also {{keep-alive-timeout}}.
+
+### Timeout for keep alive packets {#keep-alive-timeout}
+
+Name:
+: keepaliveTimeout
+
+Type:
+: Numeric, with special value `Disabled`
+
+Default:
+: Disabled
+
+A transport system can be requested a protcol that supports sending keep alive packets {{keep-alive}}.
+This property specifies the maximum time an idle connection (one for which no transport
+packets have been sent) should wait before 
+the Local Endpoint sends a keep-alive packet to the Remote Endpoint. Adjusting this Property
+will only take effect when the underlying stack supports sending keep-alive packet. The special value
+`Disabled` means that this timeout is not scheduled to happen, and does not request the
+transport system to send keep-alive packets. Guidance on setting this value is 
+provided in {{!BCP145}}.
+
+### Limit for Keepalives {#keep-alive-count}
 
+Name:
+: keepaliveCount
+
+Type:
+: Numeric, with special value `Disabled`
+
+Default:
+: 5
+
+Some protocols track responses to the sent keep-alive packets that are returned by the remote
+peer. When this function is supported, the Local Endpoint counts the number of consecutive 
+keep-alive packets for which no corresponding response is received. If this count
+equals the `keepaliveTimeout` then the Connection is aborted. This Property
+will only take effect when the underlying stack both supports sending keep-alive packets
+and the `keepaliveTimeout` is enabled. 
+
+A value greater than 1 is desired to provide robustness to loss of a keep-alive packet or its response.
+A larger value can increase the time to detect a path failure or to perform multipath fail-over {{multipath-policy}}. 
```

Comment:
Not sure if this is an issue as you only send keep-alives when your are idle otherwise, so taking more time to detect failure is not a huge issue. However, not sure if you even should start the failover when you are idle anyway or just wait until new data is coming. But that might depend on the application (and protocol e.g. if 0-RTT is supported or not). I guess  that is a good discussion to have somewhere in some draft but maybe not here but somewhere in the implementation draft?

Reply:
I'd be happy to remove line 1734...  I'll remove "A larger value can increase the time to detect a path failure or to perform multipath fail-over {{multipath-policy}}. "